src/common/config_opts.h

```diff
@@ -604,6 +604,7 @@ OPTION(osd_uuid, OPT_UUID, uuid_d())
 OPTION(osd_data, OPT_STR, "/var/lib/ceph/osd/$cluster-$id")
 OPTION(osd_journal, OPT_STR, "/var/lib/ceph/osd/$cluster-$id/journal")
 OPTION(osd_journal_size, OPT_INT, 5120)         // in mb
+OPTION(osd_journal_flush_on_shutdown, OPT_BOOL, true) // Flush journal to data store on shutdown
```

Comment:
I suggest we set it false by default. Otherwise, I'm not sure a obvious stop behavior change is ok to user. And a long stop will cause lots of ops scripts failed?